code/modules/admin/verbs/adminsay.dm

```diff
@@ -26,8 +26,9 @@
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Asay") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
 /client/proc/get_admin_say()
-	var/msg = input(src, null, "asay \"text\"") as text|null
-	cmd_admin_say(msg)
+	if(check_rights(R_ADMIN))
```

Comment:
Why not add an `else if(check_rights(R_MENTOR))` here for msay to be used so mentors can also use the F5 function

Reply:
Actually that's a good idea, I'll do that.